rfcs/0000-poa-testnet/0000-poa-testnet.md

```diff
@@ -0,0 +1,183 @@
+---
+Number: "0000"
+Category: <TBD>
+Status: <TBD>
+Author: <TBD>
+Organization: <TBD>
+Created: 2019-12-9
+---
+
+# POA testnet
+
+## Motivation
+
+The original intention of the aggron testnet is to provide a stable environment for contract development.
+
+However, the mining power on aggron is volatile (shown in [charts](https://explorer.nervos.org/aggron/charts)), since a rational miner has no incentive to mine the testnet coins continuously. If any new miner or mining pool joins, the mining difficulty rises fast. Once they leave, the testnet block produce becomes extremely slow due to a sudden drop of hashrate, and the average block time would require minutes or even longer.
```

Comment:
```suggestion
However, the mining power on aggron is volatile (shown in [charts](https://explorer.nervos.org/aggron/charts)), since a rational miner has no incentive to mine the testnet coins continuously. If any new miner or mining pool joins, the mining difficulty rises fast. Once they leave, new block generation becomes extremely slow due to a sudden drop of hashrate, and the average block time would require minutes or even longer.
```